azure-pipelines.yml

```diff
@@ -26,6 +27,7 @@ trigger:
     - release/4.*
```

Comment:
I think we can remove `dev` from the list (trigger, pr)

Reply:
done